addon/ng2/tasks/format.js

```diff
@@ -0,0 +1,23 @@
+/* jshint node: true */
+'use strict';
+
+var Promise = require('ember-cli/lib/ext/promise');
+var Task = require('ember-cli/lib/models/task');
+var exec = Promise.denodeify(require('child_process').exec);
```

Comment:
OOC, couldn't `Promise` and `exec` be imported when the task is run (so that `addon/ng2/index.js` "loads" faster) ?
(I guess they are already imported elsewhere, so it won't make any real difference, but wouldn't it theoretically be better ?)